src/crimson/os/seastore/extent_placement_manager.cc

```diff
@@ -799,7 +799,14 @@ RandomBlockOolWriter::do_write(
     ).safe_then([&t, &ex, paddr, FNAME]() {
```

Comment:
For the line 793 above (`return rbm->write(paddr, ex->get_bptr())`), isn't the goal of inplace overwrite to only write down the delta portion of the data, instead of the entire extent, to save device write bandwidth ?

Reply:
I added a new commit to address your comment.